setup.py

```diff
@@ -133,13 +144,24 @@ def cuda_info(cmd):
             return [incdir]
 
     if cmd == "libdirs":
-        for libdir in ("lib64", "lib"):
+        dirs = []
+        for libdir in ("lib64", "lib", "bin"):
```

Comment:
Why are there libraries in `bin`?

Reply:
On Windows a lot of softwares would do that. I don't know if there is a standard equivalent to the Linux filesystem hierarchy standard...

Reply:
Perhaps we should just use the one that contains the `cudart` library instead of returning all of them.

Reply:
By the way, this code comes from mpi4py IIRC (unless Dion modified it later) so you could look there for how they handle things

Reply:
Strange, I thought you took that from CuPy :)

Reply:
ops, sorry! I completely confused myself...